src/core/jsdoc-tsd-parser.ts

```diff
@@ -53,9 +54,15 @@ export class JSDocTsdParser {
 				console.log(err);
 			}
 		} else {
-			let functionBody = this.config.versionComparator.substr(this.config.versionComparator.indexOf("{") + 1);
-			functionBody = functionBody.substr(0, functionBody.length - 1).trim();
-			this.config.versionComparator = new Function("param1", "param2", functionBody);
+			if(this.config.versionComparator.indexOf("{") > 0){
+				let functionBody = this.config.versionComparator.substr(this.config.versionComparator.indexOf("{") + 1);
+				functionBody = functionBody.substr(0, functionBody.length - 1).trim();
+				this.config.versionComparator = new Function("param1", "param2", functionBody);
+			} else if (fs.statSync(this.config.versionComparator).isFile()) {

```

Comment:
Are you sure that the statSync-function doesn't throw an exception if the file doesn't exists? Or that you can use the isFilefunction in that case?